awx/ui/client/src/templates/job_templates/job-template.form.js

```diff
@@ -271,6 +271,20 @@ function(NotificationsList, i18n) {
                     },
                     ngDisabled: '!(job_template_obj.summary_fields.user_capabilities.edit || canAddJobTemplate)'
                 },
+                job_shard_count: {
+                    label: i18n._('Number of job shards to use'),
```

Comment:
This might have already been mentioned elsewhere in the review, but we don't want the word 'shard' visible to anyone. The current alternative being used is 'split'.